packages/block-library/src/navigation/style.scss

```diff
@@ -90,6 +90,14 @@
 			stroke: currentColor;
 		}
 	}
+
+	// Styles for button toggles.
+	button.wp-block-navigation__submenu-icon.wp-block-navigation__submenu-icon.wp-block-navigation__submenu-icon.wp-block-navigation__submenu-icon {
+		padding: $grid-unit $grid-unit-20;
```

Comment:
In my testing, this specificity wasn't necessary, and the added padding makes the chevron look very different from editor and frontend. I have a followup in #35030.